src/main/java/gregtech/common/metatileentities/multi/electric/MetaTileEntityProcessingArray.java

```diff
@@ -353,6 +353,18 @@ public void findMachineStack() {
             this.currentMachineStack = machine;
         }
 
+        public void updateCleanroom() {
```

Comment:
Should probably make this private. Not blocking a merge, but would be nice. Thank you for this fix!

Reply:
Done.